src/v3/src/transformer/identify/transformIdentify.ts

```diff
@@ -31,6 +31,7 @@ export const transformIdentify: IdxStepTransformer = (_, formBag, widgetProps) =
     scope: `#/properties/${ButtonType.SUBMIT}`,
     options: {
       type: ButtonType.SUBMIT,
+      dataType: 'save',
```

Comment:
https://github.com/okta/okta-signin-widget/blob/a034381ea73f538be0a47225d4d29f45fc78d23b/test/testcafe/framework/page-objects/components/BaseFormObject.js#L6

NOTE: `options.dataType` set by `getButtonControls.ts` was being overridden without this line.

https://github.com/okta/okta-signin-widget/blob/a034381ea73f538be0a47225d4d29f45fc78d23b/src/v3/src/transformer/getButtonControls.ts#L55